company-profiles/product-hackers.md

```diff
@@ -0,0 +1,33 @@
+# Product Hackers
+
+## Company blurb
+
+Product Hackers is an early company based in Madrid which develop cool software and technology, mainly web and APP. We offer consultancy services, **growth** services -which are our core- and develop in-house products to sell them worldwide.
```

Comment:
Should `-which are our core-` be italicized? 

Is so, that should be `_which are our core_`

Reply:
You are right! Fixed